src/store/metadata/stub/content/editions/average-travel-time_2023-08.json

```diff
@@ -0,0 +1,116 @@
+{
+  "@context": "https://staging.idpd.uk/ns#",
+  "@id": "https://staging.idpd.uk/datasets/average-travel-time/editions",
+  "@type": "hydra:Collection",
+  "title": "Travel time to employment centres by mode of travel",
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/average-travel-time/editions/2023-08",
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/average-travel-time",
+      "identifier": "2023-08",
+      "title": "Total value of UK exports",
+      "summary": "Average travel time in minutes to nearest employment centre with 500 to 4999 jobs available by public transport or walking, by cycle and by car (experimental statistics), England, 2014-2019.",
+      "description": "This dataset shows the average travel time in minutes to nearest employment centre with 500 to 4999 jobs available by public transport or walking, by cycle and by car (experimental statistics), in England, for periods 2014-2019.",
+      "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "creator": "ofcom",
```

Comment:
Please change this.
The creator is Dept for Transport.
You would need to include the Dept for Transport url there. Please see requested change above.